service/common/src/main/java/org/apache/polaris/service/admin/PolarisServiceImpl.java

```diff
@@ -130,7 +132,7 @@ public Response createCatalog(
   }
 
   private void validateStorageConfig(StorageConfigInfo storageConfigInfo) {
-    PolarisCallContext polarisCallContext = CallContext.getCurrentContext().getPolarisCallContext();
+    PolarisCallContext polarisCallContext = callContext.getPolarisCallContext();
```

Comment:
I suggest postponing this change until we've fixed line 101.

Reply:
Let's keep this PR as it is - there is already #463 to get rid of the problematic TLs